cloudbuild/kubectl.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
nit: the filename kubectl.sh is a little bit confusing to me. I'd prefer something like run-presubmit-on-k8s.sh.

Reply:
Done.